src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx

```diff
@@ -15,92 +15,111 @@ redirects:
   - /docs/apm/apm-ui-pages/monitoring/apm-data-infrastructure
 ---
 
-import infrastructureInfraApmDataChart from 'images/infrastructure_screenshot-crop_infra-apm-data-chart.webp'
+import apmCriticalViolationsinApdex from 'images/apm_screenshot-full_critical-violations-in-Apdex.webp'
 
-The integration of [APM](/docs/apm/new-relic-apm/getting-started/welcome-new-relic-apm) and [infrastructure](/docs/infrastructure/new-relic-infrastructure/getting-started/welcome-new-relic-infrastructure) data lets you see your APM data and infrastructure data side by side so you can find the root cause of problems more quickly.
+import apmCPUPercentageViolationinAPM from 'images/apm_screenshot-crop_CPU-percentage-violation-in-APM.webp'
 
-The main ways to find and use APM data in infrastructure monitoring are:
+import apmToggleBetweenHistogramViews from 'images/apm_screenshot-crop_toggle-between-histogram-views.webp'
 
-* [View APM charts on infrastructure monitoring UI pages](#apm-chart)
-* [Filter hosts by application data](#filter-by-app)
-* [Switch between infrastructure and APM](#switch)
-* [Examine APM data in Inventory and Events pages](#inventory-events)
+import apmExploringaSpecificHostinAPM from 'images/apm_screenshot-full_exploring-a-specific-host-in-APM.gif'
 
-Infrastructure data appears in <InlinePopover type="apm" /> in the host table on the [APM Summary](/docs/apm/applications-menu/monitoring/apm-overview-page) page.
+import apmFilterbyApplication from 'images/apm_screenshot-full_filter-by-application.webp'
 
-## View logs for your APM and infrastructure data [#logs-context]
-
-You can also bring your logs and application's data together to make troubleshooting easier and faster. With [logs in context](/docs/logs/logs-context/configure-logs-context-apm-agents/), you can see log messages related to your errors and traces directly in your app's UI. You can also see logs in context of your [infrastructure data](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/), such as Kubernetes clusters. No need to switch to another UI page.
-
-## How to integrate APM and infrastructure data [#how-it-works]
+The integration of [APM](/docs/apm/new-relic-apm/getting-started/welcome-new-relic-apm) and [infrastructure](/docs/infrastructure/new-relic-infrastructure/getting-started/welcome-new-relic-infrastructure) data allows you to see the health of your entire system from a single page. On the APM **Summary** page you can monitor hosts, apps, events, and alerts activity and use embedded change tracking to compare your data with any recent deployments. From one page you can respond to an alert, identify a root cause, and quickly resolve any impacts to host performance. 
```

Comment:
Overall this is really good value prop and a strong hook. For the final sentence, I would put the "From one page" part at the end, but overall I like this intro a lot